Add mysql & httpd docs (#1)
observIQ/ops-agent: docs/mysql.md

# `mysql` Metrics Receiver

> The mysql receiver can retrieve stats from your mysql instance using the `mod_status` endpoint.

Reviewer: ```suggestion
The mysql receiver can retrieve stats from your mysql instance by connecting as a monitoring user.
```

Author: updated

## Prerequisites

> Monitoring user must be granted the following permissions:

Reviewer: ```suggestion
It is recommended that you create a dedicated monitoring user. The monitoring user must be granted the following permissions:
```

Author: updated

```
PROCESS ON *.*
SELECT ON INFORMATION_SCHEMA.INNODB_METRICS
```

## Configuration

Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your mysql configuration.

To configure a receiver for your mysql metrics, specify the following fields:

| Field                 | Default                   | Description |
| ---                   | ---                       | ---         |
| `type`                | required                  | Must be `mysql`. |

> | `stub_status_url`     | `localhost:3306`          | The url exposed by mysql |

Reviewer: ```suggestion
| `endpoint`     | `localhost:3306`          | The url exposed by mysql |
```

Author: updated

| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
| `username`            |                           | username used to connect to the server

> | `password`            |                           | password used to connect to the server

Reviewer: For consistency, let's user complete sentences, with appropriate capitalization and punctuation.

Author: updated

Example Configuration:

```yaml
metrics:
  receivers:
    mysql_metrics:
      type: mysql

>       stub_status_url: localhost:3306

Reviewer: ```suggestion
      endpoint: localhost:3306
```

Author: updated
